test/test_transformers.py

```diff
@@ -1493,6 +1495,8 @@ def test_invalid_fused_inputs_attn_mask_present(self, device, kernel: SDPBackend
     @onlyCUDA
     @unittest.skipIf(not PLATFORM_SUPPORTS_FLASH_ATTENTION, "Does not support fused SDPA or pre-SM80 hardware")
     def test_unaligned_tensors(self, device):
+        if TEST_WITH_ROCM:  # ROCm GPU kernels does not have strict alignment requirements
+            return
```

Comment:
This comment seems contradictory to what it actually does, feels like you want to skip test with ROCM
```suggestion
```

Reply:
This UT acts in an inverted manner: it expects a RuntimeError thrown for irregular shapes to pass, otherwise fails.

However ROCm's implementation does not have such restrictions and no RuntimeError will be thrown, and thus the UT failed because the inputs are acceptable on ROCm.